### PR TITLE
Add travis yml file.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
+  - 1.11.x
+
+go_import_path: contrib.go.opencensus.io/exporter/ocagent
```
Comment (songy23) [inline .travis.yml]: contrib.go.opencensus.io/exporter/ocagent -> contrib.go.opencensus.io/resourse

Comment (bogdandrutu) [reply .travis.yml]: done.

**Review by songy23 — APPROVED** [review]
```shell
package go.opencensus.io/resource: cannot find package "go.opencensus.io/resource" in any of:
	/home/travis/.gimme/versions/go1.11.1.linux.amd64/src/go.opencensus.io/resource (from $GOROOT)
	/home/travis/gopath/src/go.opencensus.io/resource (from $GOPATH)
```

Probably need to wait a bit until the vanity url goes live.

**Comment by bogdandrutu** [comment]
Actually the problem is that the code does not compile :)

**Pull request merged**